README.md

```diff
@@ -26,45 +26,6 @@ The cluster definition file enables the following customizations to your Docker
 * [Managed Disks](examples/disks-managed) (under private preview) - shows how to use managed disks 
 * [Large Clusters](examples/largeclusters) - shows how to create cluster sizes of up to 1200 nodes
 
-## Development (Docker)
```

Comment:
Please ensure you update all documents that point to this section.  I would recommend instead leave "Development (Docker)" here and point to the new location.